feat: proto v2

ThomasRalee opened the pull request:
<!-- This is an auto-generated comment: release notes by coderabbit.ai -->

## Summary by CodeRabbit

* **Documentation**
  * Removed GitBook-hosted documentation for guides, API references, and module documentation.

* **Chores**
  * Removed beta and canary release workflows.
  * Restructured dev publishing pipeline with improved versioning and per-package publishing.
  * Removed bundle size analysis tooling.
  * Updated git hooks to include OFAC list fetching and pre-push type checking and linting.

<sub>✏️ Tip: You can customize this high-level summary in your review settings.</sub>

<!-- end of auto-generated comment: release notes by coderabbit.ai -->

ThomasRalee committed:
feat: proto v2

ThomasRalee committed:
chore: migrate ChainGrpcApis to new protobuf

bangjelkoski commented on packages/sdk-ts/test-gov-migration.js:
```diff
@@ -0,0 +1,38 @@
+// Simple test to verify Gov API migration works
+const { ChainGrpcGovApi } = require('./dist/cjs/client/chain/grpc/ChainGrpcGovApi');
```
Can we move this to a separate folder or it's a temporary script?

ThomasRalee committed:
chore: migrate IndexerGrpcApis to new the protobuf

ivan-angjelkoski commented on packages/sdk-ts/src/client/base/BaseIndexerGrpcConsumerV2.ts:
```diff
@@ -0,0 +1,11 @@
+import BaseGrpcConsumerV2 from './BaseGrpcConsumerV2.js'
```
should we delete this file?

ThomasRalee committed:
chore: migrate OLPGrpcApi to new the protobuf

ThomasRalee committed:
chore: migrate msgs to protobuf v2

ThomasRalee committed:
chore: migrate from jest to vitest

ThomasRalee committed:
chore: optimise eslint speed

ThomasRalee committed:
chore: migrate from tsc to tsdown

ivan-angjelkoski committed:
chore: update trezor version

ThomasRalee committed:
chore: remove buffer

ThomasRalee committed:
chore: revert protobufs extension from .mjs to .js - canary

ThomasRalee committed:
chore: release

 - @injectivelabs/exceptions@1.16.25-alpha.0
 - @injectivelabs/networks@1.16.25-alpha.0
 - @injectivelabs/sdk-ts@1.16.25-alpha.0
 - @injectivelabs/ts-types@1.16.25-alpha.0
 - @injectivelabs/utils@1.16.25-alpha.0
 - @injectivelabs/wallet-base@1.16.25-alpha.0
 - @injectivelabs/wallet-core@1.16.25-alpha.0
 - @injectivelabs/wallet-cosmos@1.16.25-alpha.0
 - @injectivelabs/wallet-cosmos-strategy@1.16.25-alpha.0
 - @injectivelabs/wallet-cosmostation@1.16.25-alpha.0
 - @injectivelabs/wallet-evm@1.16.25-alpha.0
 - @injectivelabs/wallet-ledger@1.16.25-alpha.0
 - @injectivelabs/wallet-magic@1.16.25-alpha.0
 - @injectivelabs/wallet-private-key@1.16.25-alpha.0
 - @injectivelabs/wallet-strategy@1.16.25-alpha.0
 - @injectivelabs/wallet-trezor@1.16.25-alpha.0
 - @injectivelabs/wallet-turnkey@1.16.25-alpha.0
 - @injectivelabs/wallet-wallet-connect@1.16.25-alpha.0

Conflicts:

ThomasRalee committed:
fix: tsconfig path mapping

ThomasRalee committed:
chore: package bump

ThomasRalee committed:
chore: release

 - @injectivelabs/exceptions@1.16.25-alpha.1
 - @injectivelabs/networks@1.16.25-alpha.1
 - @injectivelabs/sdk-ts@1.16.25-alpha.1
 - @injectivelabs/ts-types@1.16.25-alpha.1
 - @injectivelabs/utils@1.16.25-alpha.1
 - @injectivelabs/wallet-base@1.16.25-alpha.1
 - @injectivelabs/wallet-core@1.16.25-alpha.1
 - @injectivelabs/wallet-cosmos@1.16.25-alpha.1
 - @injectivelabs/wallet-cosmos-strategy@1.16.25-alpha.1
 - @injectivelabs/wallet-cosmostation@1.16.25-alpha.1
 - @injectivelabs/wallet-evm@1.16.25-alpha.1
 - @injectivelabs/wallet-ledger@1.16.25-alpha.1
 - @injectivelabs/wallet-magic@1.16.25-alpha.1
 - @injectivelabs/wallet-private-key@1.16.25-alpha.1
 - @injectivelabs/wallet-strategy@1.16.25-alpha.1
 - @injectivelabs/wallet-trezor@1.16.25-alpha.1
 - @injectivelabs/wallet-turnkey@1.16.25-alpha.1
 - @injectivelabs/wallet-wallet-connect@1.16.25-alpha.1

ThomasRalee committed:
fix: trezor - canary

ThomasRalee committed:
chore: release

 - @injectivelabs/exceptions@1.16.25-alpha.2
 - @injectivelabs/networks@1.16.25-alpha.2
 - @injectivelabs/sdk-ts@1.16.25-alpha.2
 - @injectivelabs/ts-types@1.16.25-alpha.2
 - @injectivelabs/utils@1.16.25-alpha.2
 - @injectivelabs/wallet-base@1.16.25-alpha.2
 - @injectivelabs/wallet-core@1.16.25-alpha.2
 - @injectivelabs/wallet-cosmos@1.16.25-alpha.2
 - @injectivelabs/wallet-cosmos-strategy@1.16.25-alpha.2
 - @injectivelabs/wallet-cosmostation@1.16.25-alpha.2
 - @injectivelabs/wallet-evm@1.16.25-alpha.2
 - @injectivelabs/wallet-ledger@1.16.25-alpha.2
 - @injectivelabs/wallet-magic@1.16.25-alpha.2
 - @injectivelabs/wallet-private-key@1.16.25-alpha.2
 - @injectivelabs/wallet-strategy@1.16.25-alpha.2
 - @injectivelabs/wallet-trezor@1.16.25-alpha.2
 - @injectivelabs/wallet-turnkey@1.16.25-alpha.2
 - @injectivelabs/wallet-wallet-connect@1.16.25-alpha.2

ThomasRalee committed:
fix: trezor - canary

Frederick-88 committed:
chore: add funding sum info to positions data

ivan-angjelkoski committed:
chore: minor trezor beta

ThomasRalee committed:
refactor: clean up pacakge.json and implemented pnpm catalog

ThomasRalee committed:
chore: add support for lerna caching

ThomasRalee committed:
chore: remove alchemy-sdk dependency

ThomasRalee committed:
chore: remove apollo client

bangjelkoski committed:
chore: add support for MsgCreateTokenPair

ThomasRalee committed:
chore: improve getViemPublicClient to check for alchemy url if its mainnet or sepholia chainId - canary

ThomasRalee committed:
chore: release

 - @injectivelabs/exceptions@1.16.38-alpha.0
 - @injectivelabs/networks@1.16.38-alpha.0
 - @injectivelabs/sdk-ts@1.16.38-alpha.0
 - @injectivelabs/ts-types@1.16.38-alpha.0
 - @injectivelabs/utils@1.16.38-alpha.0
 - @injectivelabs/wallet-base@1.16.38-alpha.0
 - @injectivelabs/wallet-core@1.16.38-alpha.0
 - @injectivelabs/wallet-cosmos@1.16.38-alpha.0
 - @injectivelabs/wallet-cosmos-strategy@1.16.38-alpha.0
 - @injectivelabs/wallet-cosmostation@1.16.38-alpha.0
 - @injectivelabs/wallet-evm@1.16.38-alpha.0
 - @injectivelabs/wallet-ledger@1.16.38-alpha.0
 - @injectivelabs/wallet-magic@1.16.38-alpha.0
 - @injectivelabs/wallet-private-key@1.16.38-alpha.0
 - @injectivelabs/wallet-strategy@1.16.38-alpha.0
 - @injectivelabs/wallet-trezor@1.16.38-alpha.0
 - @injectivelabs/wallet-turnkey@1.16.38-alpha.0
 - @injectivelabs/wallet-wallet-connect@1.16.38-alpha.0

ThomasRalee committed:
fix: pnpm catalog - canary

ThomasRalee committed:
chore: release

 - @injectivelabs/sdk-ts@1.16.38-alpha.1
 - @injectivelabs/wallet-base@1.16.38-alpha.1
 - @injectivelabs/wallet-core@1.16.38-alpha.1
 - @injectivelabs/wallet-cosmos@1.16.38-alpha.1
 - @injectivelabs/wallet-cosmos-strategy@1.16.38-alpha.1
 - @injectivelabs/wallet-cosmostation@1.16.38-alpha.1
 - @injectivelabs/wallet-evm@1.16.38-alpha.1
 - @injectivelabs/wallet-ledger@1.16.38-alpha.1
 - @injectivelabs/wallet-magic@1.16.38-alpha.1
 - @injectivelabs/wallet-private-key@1.16.38-alpha.1
 - @injectivelabs/wallet-strategy@1.16.38-alpha.1
 - @injectivelabs/wallet-trezor@1.16.38-alpha.1
 - @injectivelabs/wallet-turnkey@1.16.38-alpha.1
 - @injectivelabs/wallet-wallet-connect@1.16.38-alpha.1

ThomasRalee committed:
fix: pnpm catalog - canary

ThomasRalee committed:
chore: release

 - @injectivelabs/exceptions@1.16.38-alpha.1
 - @injectivelabs/networks@1.16.38-alpha.1
 - @injectivelabs/sdk-ts@1.16.38-alpha.2
 - @injectivelabs/ts-types@1.16.38-alpha.1
 - @injectivelabs/utils@1.16.38-alpha.1
 - @injectivelabs/wallet-base@1.16.38-alpha.2
 - @injectivelabs/wallet-core@1.16.38-alpha.2
 - @injectivelabs/wallet-cosmos@1.16.38-alpha.2
 - @injectivelabs/wallet-cosmos-strategy@1.16.38-alpha.2
 - @injectivelabs/wallet-cosmostation@1.16.38-alpha.2
 - @injectivelabs/wallet-evm@1.16.38-alpha.2
 - @injectivelabs/wallet-ledger@1.16.38-alpha.2
 - @injectivelabs/wallet-magic@1.16.38-alpha.2
 - @injectivelabs/wallet-private-key@1.16.38-alpha.2
 - @injectivelabs/wallet-strategy@1.16.38-alpha.2
 - @injectivelabs/wallet-trezor@1.16.38-alpha.2
 - @injectivelabs/wallet-turnkey@1.16.38-alpha.2
 - @injectivelabs/wallet-wallet-connect@1.16.38-alpha.2

ivan-angjelkoski committed:
chore: update ledger versions

ThomasRalee committed:
chore: add support for keplr-evm wallet

ThomasRalee committed:
chore: update wallet connect dependency

ThomasRalee committed:
chore: tree-shake wallet strategy

ThomasRalee committed:
chore: sdk-ts subpath imports - canary

ThomasRalee committed:
chore: release

 - @injectivelabs/exceptions@1.16.38-alpha.2
 - @injectivelabs/sdk-ts@1.16.38-alpha.3
 - @injectivelabs/utils@1.16.38-alpha.2
 - @injectivelabs/wallet-base@1.16.38-alpha.3
 - @injectivelabs/wallet-core@1.16.38-alpha.3
 - @injectivelabs/wallet-cosmos@1.16.38-alpha.3
 - @injectivelabs/wallet-cosmos-strategy@1.16.38-alpha.3
 - @injectivelabs/wallet-cosmostation@1.16.38-alpha.3
 - @injectivelabs/wallet-evm@1.16.38-alpha.3
 - @injectivelabs/wallet-ledger@1.16.38-alpha.3
 - @injectivelabs/wallet-magic@1.16.38-alpha.3
 - @injectivelabs/wallet-private-key@1.16.38-alpha.3
 - @injectivelabs/wallet-strategy@1.16.38-alpha.3
 - @injectivelabs/wallet-trezor@1.16.38-alpha.3
 - @injectivelabs/wallet-turnkey@1.16.38-alpha.3
 - @injectivelabs/wallet-wallet-connect@1.16.38-alpha.3

ivan-angjelkoski committed:
chore: hw wallet derivation path metadata

ivan-angjelkoski committed:
feat: trezor eip1193 provider

ThomasRalee committed:
fix: subpath configuration

ThomasRalee committed:
chore: update dependencies version

ivan-angjelkoski committed:
chore: minor address info

ivan-angjelkoski committed:
chore: minor canary

ivan-angjelkoski committed:
chore: minor canary

ivan-angjelkoski committed:
chore: minor canary

ivan-angjelkoski committed:
chore: release

 - @injectivelabs/exceptions@1.16.38-alpha.3
 - @injectivelabs/networks@1.16.38-alpha.2
 - @injectivelabs/sdk-ts@1.16.38-alpha.4
 - @injectivelabs/ts-types@1.16.38-alpha.2
 - @injectivelabs/utils@1.16.38-alpha.3
 - @injectivelabs/wallet-base@1.16.38-alpha.4
 - @injectivelabs/wallet-core@1.16.38-alpha.4
 - @injectivelabs/wallet-cosmos@1.16.38-alpha.4
 - @injectivelabs/wallet-cosmos-strategy@1.16.38-alpha.4
 - @injectivelabs/wallet-cosmostation@1.16.38-alpha.4
 - @injectivelabs/wallet-evm@1.16.38-alpha.4
 - @injectivelabs/wallet-ledger@1.16.38-alpha.4
 - @injectivelabs/wallet-magic@1.16.38-alpha.4
 - @injectivelabs/wallet-private-key@1.16.38-alpha.4
 - @injectivelabs/wallet-strategy@1.16.38-alpha.4
 - @injectivelabs/wallet-trezor@1.16.38-alpha.4
 - @injectivelabs/wallet-turnkey@1.16.38-alpha.4
 - @injectivelabs/wallet-wallet-connect@1.16.38-alpha.4

ivan-angjelkoski committed:
fix: imports proto v2 canary

ivan-angjelkoski committed:
chore: release

 - @injectivelabs/sdk-ts@1.16.38-alpha.5
 - @injectivelabs/utils@1.16.38-alpha.4
 - @injectivelabs/wallet-base@1.16.38-alpha.5
 - @injectivelabs/wallet-core@1.16.38-alpha.5
 - @injectivelabs/wallet-cosmos@1.16.38-alpha.5
 - @injectivelabs/wallet-cosmos-strategy@1.16.38-alpha.5
 - @injectivelabs/wallet-cosmostation@1.16.38-alpha.5
 - @injectivelabs/wallet-evm@1.16.38-alpha.5
 - @injectivelabs/wallet-ledger@1.16.38-alpha.5
 - @injectivelabs/wallet-magic@1.16.38-alpha.5
 - @injectivelabs/wallet-private-key@1.16.38-alpha.5
 - @injectivelabs/wallet-strategy@1.16.38-alpha.5
 - @injectivelabs/wallet-trezor@1.16.38-alpha.5
 - @injectivelabs/wallet-turnkey@1.16.38-alpha.5
 - @injectivelabs/wallet-wallet-connect@1.16.38-alpha.5

ThomasRalee committed:
fix: subpath imports - canary

ThomasRalee committed:
chore: release

 - @injectivelabs/exceptions@1.16.38-alpha.4
 - @injectivelabs/networks@1.16.38-alpha.3
 - @injectivelabs/sdk-ts@1.16.38-alpha.6
 - @injectivelabs/ts-types@1.16.38-alpha.3
 - @injectivelabs/utils@1.16.38-alpha.5
 - @injectivelabs/wallet-base@1.16.38-alpha.6
 - @injectivelabs/wallet-core@1.16.38-alpha.6
 - @injectivelabs/wallet-cosmos@1.16.38-alpha.6
 - @injectivelabs/wallet-cosmos-strategy@1.16.38-alpha.6
 - @injectivelabs/wallet-cosmostation@1.16.38-alpha.6
 - @injectivelabs/wallet-evm@1.16.38-alpha.6
 - @injectivelabs/wallet-ledger@1.16.38-alpha.6
 - @injectivelabs/wallet-magic@1.16.38-alpha.6
 - @injectivelabs/wallet-private-key@1.16.38-alpha.6
 - @injectivelabs/wallet-strategy@1.16.38-alpha.6
 - @injectivelabs/wallet-trezor@1.16.38-alpha.6
 - @injectivelabs/wallet-turnkey@1.16.38-alpha.6
 - @injectivelabs/wallet-wallet-connect@1.16.38-alpha.6

Frederick-88 committed:
chore: archiver accountstats endpoint change - canary

ThomasRalee committed:
chore: rebase dev - canary

ThomasRalee committed:
chore: release

 - @injectivelabs/exceptions@1.16.39-alpha.0
 - @injectivelabs/networks@1.16.39-alpha.0
 - @injectivelabs/sdk-ts@1.16.39-alpha.0
 - @injectivelabs/ts-types@1.16.39-alpha.0
 - @injectivelabs/utils@1.16.39-alpha.0
 - @injectivelabs/wallet-base@1.16.39-alpha.0
 - @injectivelabs/wallet-core@1.16.39-alpha.0
 - @injectivelabs/wallet-cosmos@1.16.39-alpha.0
 - @injectivelabs/wallet-cosmos-strategy@1.16.39-alpha.0
 - @injectivelabs/wallet-cosmostation@1.16.39-alpha.0
 - @injectivelabs/wallet-evm@1.16.39-alpha.0
 - @injectivelabs/wallet-ledger@1.16.39-alpha.0
 - @injectivelabs/wallet-magic@1.16.39-alpha.0
 - @injectivelabs/wallet-private-key@1.16.39-alpha.0
 - @injectivelabs/wallet-strategy@1.16.39-alpha.0
 - @injectivelabs/wallet-trezor@1.16.39-alpha.0
 - @injectivelabs/wallet-turnkey@1.16.39-alpha.0
 - @injectivelabs/wallet-wallet-connect@1.16.39-alpha.0

ThomasRalee committed:
fix: msgSubmitProposalPerpetualMarketLaunchV2 missing open_notional_cap field - canary

ThomasRalee committed:
chore: release

 - @injectivelabs/sdk-ts@1.16.39-alpha.1
 - @injectivelabs/utils@1.16.39-alpha.1
 - @injectivelabs/wallet-base@1.16.39-alpha.1
 - @injectivelabs/wallet-core@1.16.39-alpha.1
 - @injectivelabs/wallet-cosmos@1.16.39-alpha.1
 - @injectivelabs/wallet-cosmos-strategy@1.16.39-alpha.1
 - @injectivelabs/wallet-cosmostation@1.16.39-alpha.1
 - @injectivelabs/wallet-evm@1.16.39-alpha.1
 - @injectivelabs/wallet-ledger@1.16.39-alpha.1
 - @injectivelabs/wallet-magic@1.16.39-alpha.1
 - @injectivelabs/wallet-private-key@1.16.39-alpha.1
 - @injectivelabs/wallet-strategy@1.16.39-alpha.1
 - @injectivelabs/wallet-trezor@1.16.39-alpha.1
 - @injectivelabs/wallet-turnkey@1.16.39-alpha.1
 - @injectivelabs/wallet-wallet-connect@1.16.39-alpha.1

ThomasRalee committed:
chore: enable force_optimize_code_size on protov2 gen.sh script

ivan-angjelkoski committed:
chore: derivation path hw canary

ivan-angjelkoski committed:
chore: release

 - @injectivelabs/exceptions@1.17.2-alpha.0
 - @injectivelabs/networks@1.17.2-alpha.0
 - @injectivelabs/sdk-ts@1.17.2-alpha.0
 - @injectivelabs/ts-types@1.17.2-alpha.0
 - @injectivelabs/utils@1.17.2-alpha.0
 - @injectivelabs/wallet-base@1.17.2-alpha.0
 - @injectivelabs/wallet-core@1.17.2-alpha.0
 - @injectivelabs/wallet-cosmos@1.17.2-alpha.0
 - @injectivelabs/wallet-cosmos-strategy@1.17.2-alpha.0
 - @injectivelabs/wallet-cosmostation@1.17.2-alpha.0
 - @injectivelabs/wallet-evm@1.17.2-alpha.0
 - @injectivelabs/wallet-ledger@1.17.2-alpha.0
 - @injectivelabs/wallet-magic@1.17.2-alpha.0
 - @injectivelabs/wallet-private-key@1.17.2-alpha.0
 - @injectivelabs/wallet-strategy@1.17.2-alpha.0
 - @injectivelabs/wallet-trezor@1.17.2-alpha.0
 - @injectivelabs/wallet-turnkey@1.17.2-alpha.0
 - @injectivelabs/wallet-wallet-connect@1.17.2-alpha.0

ThomasRalee committed:
fix: temporal dead zone issue - canary

ThomasRalee committed:
chore: release

 - @injectivelabs/exceptions@1.17.2-alpha.1
 - @injectivelabs/networks@1.17.2-alpha.1
 - @injectivelabs/sdk-ts@1.17.2-alpha.1
 - @injectivelabs/ts-types@1.17.2-alpha.1
 - @injectivelabs/utils@1.17.2-alpha.1
 - @injectivelabs/wallet-base@1.17.2-alpha.1
 - @injectivelabs/wallet-core@1.17.2-alpha.1
 - @injectivelabs/wallet-cosmos@1.17.2-alpha.1
 - @injectivelabs/wallet-cosmos-strategy@1.17.2-alpha.1
 - @injectivelabs/wallet-cosmostation@1.17.2-alpha.1
 - @injectivelabs/wallet-evm@1.17.2-alpha.1
 - @injectivelabs/wallet-ledger@1.17.2-alpha.1
 - @injectivelabs/wallet-magic@1.17.2-alpha.1
 - @injectivelabs/wallet-private-key@1.17.2-alpha.1
 - @injectivelabs/wallet-strategy@1.17.2-alpha.1
 - @injectivelabs/wallet-trezor@1.17.2-alpha.1
 - @injectivelabs/wallet-turnkey@1.17.2-alpha.1
 - @injectivelabs/wallet-wallet-connect@1.17.2-alpha.1

ThomasRalee committed:
chore: extend wallet-cosmos-strategy to add support for wallet-cosmostation

ThomasRalee committed:
chore: wallet-ledger inject polyfill through tsdown - canary

ThomasRalee committed:
chore: release

 - @injectivelabs/exceptions@1.17.2-alpha.2
 - @injectivelabs/networks@1.17.2-alpha.2
 - @injectivelabs/sdk-ts@1.17.2-alpha.2
 - @injectivelabs/ts-types@1.17.2-alpha.2
 - @injectivelabs/utils@1.17.2-alpha.2
 - @injectivelabs/wallet-base@1.17.2-alpha.2
 - @injectivelabs/wallet-core@1.17.2-alpha.2
 - @injectivelabs/wallet-cosmos@1.17.2-alpha.2
 - @injectivelabs/wallet-cosmos-strategy@1.17.2-alpha.2
 - @injectivelabs/wallet-cosmostation@1.17.2-alpha.2
 - @injectivelabs/wallet-evm@1.17.2-alpha.2
 - @injectivelabs/wallet-ledger@1.17.2-alpha.2
 - @injectivelabs/wallet-magic@1.17.2-alpha.2
 - @injectivelabs/wallet-private-key@1.17.2-alpha.2
 - @injectivelabs/wallet-strategy@1.17.2-alpha.2
 - @injectivelabs/wallet-trezor@1.17.2-alpha.2
 - @injectivelabs/wallet-turnkey@1.17.2-alpha.2
 - @injectivelabs/wallet-wallet-connect@1.17.2-alpha.2

ThomasRalee committed:
chore: add caching for cosmjs package - canary

ThomasRalee committed:
chore: release

 - @injectivelabs/exceptions@1.17.2-alpha.3
 - @injectivelabs/networks@1.17.2-alpha.3
 - @injectivelabs/sdk-ts@1.17.2-alpha.3
 - @injectivelabs/ts-types@1.17.2-alpha.3
 - @injectivelabs/utils@1.17.2-alpha.3
 - @injectivelabs/wallet-base@1.17.2-alpha.3
 - @injectivelabs/wallet-core@1.17.2-alpha.3
 - @injectivelabs/wallet-cosmos@1.17.2-alpha.3
 - @injectivelabs/wallet-cosmos-strategy@1.17.2-alpha.3
 - @injectivelabs/wallet-cosmostation@1.17.2-alpha.3
 - @injectivelabs/wallet-evm@1.17.2-alpha.3
 - @injectivelabs/wallet-ledger@1.17.2-alpha.3
 - @injectivelabs/wallet-magic@1.17.2-alpha.3
 - @injectivelabs/wallet-private-key@1.17.2-alpha.3
 - @injectivelabs/wallet-strategy@1.17.2-alpha.3
 - @injectivelabs/wallet-trezor@1.17.2-alpha.3
 - @injectivelabs/wallet-turnkey@1.17.2-alpha.3
 - @injectivelabs/wallet-wallet-connect@1.17.2-alpha.3

ThomasRalee committed:
chore: replace crypto-js dependency with noble/hashes - canary

ThomasRalee committed:
chore: release

 - @injectivelabs/exceptions@1.17.2-alpha.4
 - @injectivelabs/networks@1.17.2-alpha.4
 - @injectivelabs/sdk-ts@1.17.2-alpha.4
 - @injectivelabs/ts-types@1.17.2-alpha.4
 - @injectivelabs/utils@1.17.2-alpha.4
 - @injectivelabs/wallet-base@1.17.2-alpha.4
 - @injectivelabs/wallet-core@1.17.2-alpha.4
 - @injectivelabs/wallet-cosmos@1.17.2-alpha.4
 - @injectivelabs/wallet-cosmos-strategy@1.17.2-alpha.4
 - @injectivelabs/wallet-cosmostation@1.17.2-alpha.4
 - @injectivelabs/wallet-evm@1.17.2-alpha.4
 - @injectivelabs/wallet-ledger@1.17.2-alpha.4
 - @injectivelabs/wallet-magic@1.17.2-alpha.4
 - @injectivelabs/wallet-private-key@1.17.2-alpha.4
 - @injectivelabs/wallet-strategy@1.17.2-alpha.4
 - @injectivelabs/wallet-trezor@1.17.2-alpha.4
 - @injectivelabs/wallet-turnkey@1.17.2-alpha.4
 - @injectivelabs/wallet-wallet-connect@1.17.2-alpha.4

ThomasRalee committed:
refactor: wallet-connect to use reown/appkit

ThomasRalee committed:
chore: setup OIDC npm token for CI package bump - canary

ThomasRalee committed:
chore: release

 - @injectivelabs/exceptions@1.17.2-alpha.5
 - @injectivelabs/networks@1.17.2-alpha.5
 - @injectivelabs/sdk-ts@1.17.2-alpha.5
 - @injectivelabs/ts-types@1.17.2-alpha.5
 - @injectivelabs/utils@1.17.2-alpha.5
 - @injectivelabs/wallet-base@1.17.2-alpha.5
 - @injectivelabs/wallet-core@1.17.2-alpha.5
 - @injectivelabs/wallet-cosmos@1.17.2-alpha.5
 - @injectivelabs/wallet-cosmos-strategy@1.17.2-alpha.5
 - @injectivelabs/wallet-cosmostation@1.17.2-alpha.5
 - @injectivelabs/wallet-evm@1.17.2-alpha.5
 - @injectivelabs/wallet-ledger@1.17.2-alpha.5
 - @injectivelabs/wallet-magic@1.17.2-alpha.5
 - @injectivelabs/wallet-private-key@1.17.2-alpha.5
 - @injectivelabs/wallet-strategy@1.17.2-alpha.5
 - @injectivelabs/wallet-trezor@1.17.2-alpha.5
 - @injectivelabs/wallet-turnkey@1.17.2-alpha.5
 - @injectivelabs/wallet-wallet-connect@1.17.2-alpha.5

ThomasRalee committed:
fix: slack broadcasting - canary

ThomasRalee committed:
chore: release

 - @injectivelabs/exceptions@1.17.2-alpha.6
 - @injectivelabs/networks@1.17.2-alpha.6
 - @injectivelabs/sdk-ts@1.17.2-alpha.6
 - @injectivelabs/ts-types@1.17.2-alpha.6
 - @injectivelabs/utils@1.17.2-alpha.6
 - @injectivelabs/wallet-base@1.17.2-alpha.6
 - @injectivelabs/wallet-core@1.17.2-alpha.6
 - @injectivelabs/wallet-cosmos@1.17.2-alpha.6
 - @injectivelabs/wallet-cosmos-strategy@1.17.2-alpha.6
 - @injectivelabs/wallet-cosmostation@1.17.2-alpha.6
 - @injectivelabs/wallet-evm@1.17.2-alpha.6
 - @injectivelabs/wallet-ledger@1.17.2-alpha.6
 - @injectivelabs/wallet-magic@1.17.2-alpha.6
 - @injectivelabs/wallet-private-key@1.17.2-alpha.6
 - @injectivelabs/wallet-strategy@1.17.2-alpha.6
 - @injectivelabs/wallet-trezor@1.17.2-alpha.6
 - @injectivelabs/wallet-turnkey@1.17.2-alpha.6
 - @injectivelabs/wallet-wallet-connect@1.17.2-alpha.6

ThomasRalee committed:
fix: canary deployment - canary

ThomasRalee committed:
chore: release

 - @injectivelabs/exceptions@1.17.2-alpha.7
 - @injectivelabs/networks@1.17.2-alpha.7
 - @injectivelabs/sdk-ts@1.17.2-alpha.7
 - @injectivelabs/ts-types@1.17.2-alpha.7
 - @injectivelabs/utils@1.17.2-alpha.7
 - @injectivelabs/wallet-base@1.17.2-alpha.7
 - @injectivelabs/wallet-core@1.17.2-alpha.7
 - @injectivelabs/wallet-cosmos@1.17.2-alpha.7
 - @injectivelabs/wallet-cosmos-strategy@1.17.2-alpha.7
 - @injectivelabs/wallet-cosmostation@1.17.2-alpha.7
 - @injectivelabs/wallet-evm@1.17.2-alpha.7
 - @injectivelabs/wallet-ledger@1.17.2-alpha.7
 - @injectivelabs/wallet-magic@1.17.2-alpha.7
 - @injectivelabs/wallet-private-key@1.17.2-alpha.7
 - @injectivelabs/wallet-strategy@1.17.2-alpha.7
 - @injectivelabs/wallet-trezor@1.17.2-alpha.7
 - @injectivelabs/wallet-turnkey@1.17.2-alpha.7
 - @injectivelabs/wallet-wallet-connect@1.17.2-alpha.7

ThomasRalee committed:
fix: buffer issue, updating bip39 to use scure/bip32 - canary

ThomasRalee committed:
chore: release

 - @injectivelabs/exceptions@1.17.2-alpha.8
 - @injectivelabs/networks@1.17.2-alpha.8
 - @injectivelabs/sdk-ts@1.17.2-alpha.8
 - @injectivelabs/ts-types@1.17.2-alpha.8
 - @injectivelabs/utils@1.17.2-alpha.8
 - @injectivelabs/wallet-base@1.17.2-alpha.8
 - @injectivelabs/wallet-core@1.17.2-alpha.8
 - @injectivelabs/wallet-cosmos@1.17.2-alpha.8
 - @injectivelabs/wallet-cosmos-strategy@1.17.2-alpha.8
 - @injectivelabs/wallet-cosmostation@1.17.2-alpha.8
 - @injectivelabs/wallet-evm@1.17.2-alpha.8
 - @injectivelabs/wallet-ledger@1.17.2-alpha.8
 - @injectivelabs/wallet-magic@1.17.2-alpha.8
 - @injectivelabs/wallet-private-key@1.17.2-alpha.8
 - @injectivelabs/wallet-strategy@1.17.2-alpha.8
 - @injectivelabs/wallet-trezor@1.17.2-alpha.8
 - @injectivelabs/wallet-turnkey@1.17.2-alpha.8
 - @injectivelabs/wallet-wallet-connect@1.17.2-alpha.8

ThomasRalee committed:
fix: bigInt seralization - canary

ThomasRalee committed:
fix: ci

ThomasRalee committed:
fix: ci

ThomasRalee committed:
chore: remove force_optimize_code_size in protobuf bundle

ThomasRalee committed:
chore: clean-up duplicated typings

ThomasRalee committed:
feat: streamManagerV2 - canary

ThomasRalee committed:
chore: release

 - @injectivelabs/exceptions@1.17.2-alpha.10
 - @injectivelabs/networks@1.17.2-alpha.10
 - @injectivelabs/sdk-ts@1.17.2-alpha.10
 - @injectivelabs/ts-types@1.17.2-alpha.10
 - @injectivelabs/utils@1.17.2-alpha.10
 - @injectivelabs/wallet-base@1.17.2-alpha.10
 - @injectivelabs/wallet-core@1.17.2-alpha.10
 - @injectivelabs/wallet-cosmos@1.17.2-alpha.10
 - @injectivelabs/wallet-cosmos-strategy@1.17.2-alpha.10
 - @injectivelabs/wallet-cosmostation@1.17.2-alpha.10
 - @injectivelabs/wallet-evm@1.17.2-alpha.10
 - @injectivelabs/wallet-ledger@1.17.2-alpha.10
 - @injectivelabs/wallet-magic@1.17.2-alpha.10
 - @injectivelabs/wallet-private-key@1.17.2-alpha.10
 - @injectivelabs/wallet-strategy@1.17.2-alpha.10
 - @injectivelabs/wallet-trezor@1.17.2-alpha.10
 - @injectivelabs/wallet-turnkey@1.17.2-alpha.10
 - @injectivelabs/wallet-wallet-connect@1.17.2-alpha.10

ThomasRalee committed:
fix: typo in sdk-ts package.json - canary

ThomasRalee committed:
chore: release

 - @injectivelabs/exceptions@1.17.2-alpha.11
 - @injectivelabs/networks@1.17.2-alpha.11
 - @injectivelabs/sdk-ts@1.17.2-alpha.11
 - @injectivelabs/ts-types@1.17.2-alpha.11
 - @injectivelabs/utils@1.17.2-alpha.11
 - @injectivelabs/wallet-base@1.17.2-alpha.11
 - @injectivelabs/wallet-core@1.17.2-alpha.11
 - @injectivelabs/wallet-cosmos@1.17.2-alpha.11
 - @injectivelabs/wallet-cosmos-strategy@1.17.2-alpha.11
 - @injectivelabs/wallet-cosmostation@1.17.2-alpha.11
 - @injectivelabs/wallet-evm@1.17.2-alpha.11
 - @injectivelabs/wallet-ledger@1.17.2-alpha.11
 - @injectivelabs/wallet-magic@1.17.2-alpha.11
 - @injectivelabs/wallet-private-key@1.17.2-alpha.11
 - @injectivelabs/wallet-strategy@1.17.2-alpha.11
 - @injectivelabs/wallet-trezor@1.17.2-alpha.11
 - @injectivelabs/wallet-turnkey@1.17.2-alpha.11
 - @injectivelabs/wallet-wallet-connect@1.17.2-alpha.11

ThomasRalee committed:
chore: streamManagerV2 remove getStats function - canary

ThomasRalee committed:
chore: release

 - @injectivelabs/exceptions@1.17.2-alpha.12
 - @injectivelabs/networks@1.17.2-alpha.12
 - @injectivelabs/sdk-ts@1.17.2-alpha.12
 - @injectivelabs/ts-types@1.17.2-alpha.12
 - @injectivelabs/utils@1.17.2-alpha.12
 - @injectivelabs/wallet-base@1.17.2-alpha.12
 - @injectivelabs/wallet-core@1.17.2-alpha.12
 - @injectivelabs/wallet-cosmos@1.17.2-alpha.12
 - @injectivelabs/wallet-cosmos-strategy@1.17.2-alpha.12
 - @injectivelabs/wallet-cosmostation@1.17.2-alpha.12
 - @injectivelabs/wallet-evm@1.17.2-alpha.12
 - @injectivelabs/wallet-ledger@1.17.2-alpha.12
 - @injectivelabs/wallet-magic@1.17.2-alpha.12
 - @injectivelabs/wallet-private-key@1.17.2-alpha.12
 - @injectivelabs/wallet-strategy@1.17.2-alpha.12
 - @injectivelabs/wallet-trezor@1.17.2-alpha.12
 - @injectivelabs/wallet-turnkey@1.17.2-alpha.12
 - @injectivelabs/wallet-wallet-connect@1.17.2-alpha.12

ivan-angjelkoski committed:
fix: ledger exception message

ThomasRalee committed:
Merge branch 'feat/proto-v2' into chore/add-funding-sum-to-positions

ThomasRalee committed:
Merge pull request #606 from InjectiveLabs/chore/add-funding-sum-to-positions

chore/add-funding-sum-to-positions

ThomasRalee committed:
chore: positon cumulative funding - canary

ThomasRalee committed:
chore: release

 - @injectivelabs/exceptions@1.17.2-alpha.13
 - @injectivelabs/networks@1.17.2-alpha.13
 - @injectivelabs/sdk-ts@1.17.2-alpha.13
 - @injectivelabs/ts-types@1.17.2-alpha.13
 - @injectivelabs/utils@1.17.2-alpha.13
 - @injectivelabs/wallet-base@1.17.2-alpha.13
 - @injectivelabs/wallet-core@1.17.2-alpha.13
 - @injectivelabs/wallet-cosmos@1.17.2-alpha.13
 - @injectivelabs/wallet-cosmos-strategy@1.17.2-alpha.13
 - @injectivelabs/wallet-cosmostation@1.17.2-alpha.13
 - @injectivelabs/wallet-evm@1.17.2-alpha.13
 - @injectivelabs/wallet-ledger@1.17.2-alpha.13
 - @injectivelabs/wallet-magic@1.17.2-alpha.13
 - @injectivelabs/wallet-private-key@1.17.2-alpha.13
 - @injectivelabs/wallet-strategy@1.17.2-alpha.13
 - @injectivelabs/wallet-trezor@1.17.2-alpha.13
 - @injectivelabs/wallet-turnkey@1.17.2-alpha.13
 - @injectivelabs/wallet-wallet-connect@1.17.2-alpha.13

ivan-angjelkoski committed:
chore: fix error msg mapping

Frederick-88 committed:
fix: issue in autosign with spot limit order chase

ivan-angjelkoski committed:
chore: dynamic error message extraction

ThomasRalee committed:
Merge pull request #610 from InjectiveLabs/chore/error-message-fix

Chore/error message fix

ivan-angjelkoski committed:
chore: publish - canary

ivan-angjelkoski committed:
chore: release

 - @injectivelabs/exceptions@1.17.2-alpha.14
 - @injectivelabs/networks@1.17.2-alpha.14
 - @injectivelabs/sdk-ts@1.17.2-alpha.14
 - @injectivelabs/ts-types@1.17.2-alpha.14
 - @injectivelabs/utils@1.17.2-alpha.14
 - @injectivelabs/wallet-base@1.17.2-alpha.14
 - @injectivelabs/wallet-core@1.17.2-alpha.14
 - @injectivelabs/wallet-cosmos@1.17.2-alpha.14
 - @injectivelabs/wallet-cosmos-strategy@1.17.2-alpha.14
 - @injectivelabs/wallet-cosmostation@1.17.2-alpha.14
 - @injectivelabs/wallet-evm@1.17.2-alpha.14
 - @injectivelabs/wallet-ledger@1.17.2-alpha.14
 - @injectivelabs/wallet-magic@1.17.2-alpha.14
 - @injectivelabs/wallet-private-key@1.17.2-alpha.14
 - @injectivelabs/wallet-strategy@1.17.2-alpha.14
 - @injectivelabs/wallet-trezor@1.17.2-alpha.14
 - @injectivelabs/wallet-turnkey@1.17.2-alpha.14
 - @injectivelabs/wallet-wallet-connect@1.17.2-alpha.14

ThomasRalee committed:
chore: msgFundCommunityPool add support to accept singular coin payload (object and array)

ThomasRalee committed:
chore: enhance streamMangerV2

ivan-angjelkoski committed:
chore: fix ledger payload size signing - canary

ivan-angjelkoski committed:
chore: release

 - @injectivelabs/exceptions@1.17.2-alpha.15
 - @injectivelabs/networks@1.17.2-alpha.15
 - @injectivelabs/sdk-ts@1.17.2-alpha.15
 - @injectivelabs/ts-types@1.17.2-alpha.15
 - @injectivelabs/utils@1.17.2-alpha.15
 - @injectivelabs/wallet-base@1.17.2-alpha.15
 - @injectivelabs/wallet-core@1.17.2-alpha.15
 - @injectivelabs/wallet-cosmos@1.17.2-alpha.15
 - @injectivelabs/wallet-cosmos-strategy@1.17.2-alpha.15
 - @injectivelabs/wallet-cosmostation@1.17.2-alpha.15
 - @injectivelabs/wallet-evm@1.17.2-alpha.15
 - @injectivelabs/wallet-ledger@1.17.2-alpha.15
 - @injectivelabs/wallet-magic@1.17.2-alpha.15
 - @injectivelabs/wallet-private-key@1.17.2-alpha.15
 - @injectivelabs/wallet-strategy@1.17.2-alpha.15
 - @injectivelabs/wallet-trezor@1.17.2-alpha.15
 - @injectivelabs/wallet-turnkey@1.17.2-alpha.15
 - @injectivelabs/wallet-wallet-connect@1.17.2-alpha.15

ThomasRalee committed:
chore: downgrade turnkey-wallet sdk version to 5.2.3 - canary

ThomasRalee committed:
chore: release

 - @injectivelabs/exceptions@1.17.2-alpha.16
 - @injectivelabs/networks@1.17.2-alpha.16
 - @injectivelabs/sdk-ts@1.17.2-alpha.16
 - @injectivelabs/ts-types@1.17.2-alpha.16
 - @injectivelabs/utils@1.17.2-alpha.16
 - @injectivelabs/wallet-base@1.17.2-alpha.16
 - @injectivelabs/wallet-core@1.17.2-alpha.16
 - @injectivelabs/wallet-cosmos@1.17.2-alpha.16
 - @injectivelabs/wallet-cosmos-strategy@1.17.2-alpha.16
 - @injectivelabs/wallet-cosmostation@1.17.2-alpha.16
 - @injectivelabs/wallet-evm@1.17.2-alpha.16
 - @injectivelabs/wallet-ledger@1.17.2-alpha.16
 - @injectivelabs/wallet-magic@1.17.2-alpha.16
 - @injectivelabs/wallet-private-key@1.17.2-alpha.16
 - @injectivelabs/wallet-strategy@1.17.2-alpha.16
 - @injectivelabs/wallet-trezor@1.17.2-alpha.16
 - @injectivelabs/wallet-turnkey@1.17.2-alpha.16
 - @injectivelabs/wallet-wallet-connect@1.17.2-alpha.16

ThomasRalee committed:
fix: typing
